bin/generate_jobs_yaml.sh

```diff
@@ -13,147 +13,241 @@ cd "$REPO"
 
 # Here are the supported nginx tags.
```

Comment:
I suppose they haven't all been "nginx tags" for a while. Might as well fix this now:
```suggestion
# Here are the supported build/test configurations.
```